_static/version

```diff
@@ -1 +1 @@
-2022.1.0
\ No newline at end of file
+2022.1.1
```

Comment:
There seems to be something wrong with the PR (it looks like the branch was created from the `current` branch). Can you rebase it onto `next` and reapply the changes? Thanks!